Casamari/MNC007.xml

```diff
@@ -128,136 +223,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                           page of <ref target="#ms_i1"/>, initial two lines of <ref target="#ms_i2"
+                           />; initial words of all other texts and of each stanza; holy names (each
+                           character alternates with black ink in the name of God on <locus
+                              target="#21r"/>); names of the individuals related to the production
+                           and ownership of the manuscript; partly elements of the punctuation
+                           marks, text dividers, and numerals.</seg>
+                        <list type="abbreviations">
+                           <item>
+                              <abbr>ሰ፡ or ሰላ፡</abbr> for <expan>ሰላም፡</expan>
+                           </item>
+                        </list>
                      </handNote>
                   </handDesc>
+
                   <decoDesc>
                      <decoNote type="band" xml:id="d1">
-                        <locus target="#3r"/>
-                        <desc>Coloured ornamental band with simple drawings</desc>
+                        <desc>An ornamental band in black and red ink with a face is drawn on <locus
+                              target="#3r"/> (incipit page of <ref target="#ms_i1"/>).</desc>
                      </decoNote>
                   </decoDesc>
+
                   <additions>
                      <list>
                         <item xml:id="a1">
-                           <desc type="Unclear"/>
-                           <locus target="#1v"/>
-                           <desc>Unclear note written in Amharic.
-                           </desc>
-                        </item> 
+                           <locus from="41v" to="42r"/>
+                           <desc type="GuestText">
+                              <title ref="LIT5888AsmatPrayer"><foreign xml:lang="gez"
+                                    >ʾAsmāt</foreign>-prayer</title>. The prayer is similar to the
+                              shorter one contained in <ref type="mss" corresp="EMIP02011"/>. The
+                              prayer was written by two scribes. The first started writing with
+                              bulky letters and stopped after a few lines. The remaining text was written by a more skilled hand in very small
+                              characters.</desc>
+                           <note/>
+                           <q xml:lang="gez">
+                              <seg part="I">
+                                 <locus target="#41v"/>ቶቤር፡ ፯፡ አለበዎ፡ ቃለ፡ ድርሰት፡ ወት<supplied
+                                    reason="omitted" resp="SH">ር</supplied>ጓሜ፡ ወይትረኃዋ፡ ሎቱ፡ አናቅ<del
+                                    rend="expunctuated" cause="superfluous">ፅ</del>ጸ፡ ሰማያት፡ ከመ፡ ማይ፡
+                                 አሩጽ፡ ቃለ፡ ድርሰት፡ ወት<supplied reason="omitted" resp="SH"
+                                 >ር</supplied>ጓሜ፡ <sic>ኵ</sic>፩፡ መጻሕ<supplied reason="omitted"
```

Comment:
Sanne, here I think that also the characters: ኵ and ፩፡ are expunctuated from the text

Reply:
Sorry; I missed this comment. ኵ is expunctuated; it looks like marking the numeral ፩. The numeral ፩ is correct.
I will add the mark up for ኵ.